bgpd/bgp_fsm.c

```diff
@@ -439,6 +439,17 @@ bgp_stop (struct peer *peer)
   safi_t safi;
   char orf_name[BUFSIZ];
 
+  if (peer_dynamic_neighbor(peer) &&
+      !(CHECK_FLAG(peer->flags, PEER_FLAG_DELETE)))
+    {
+      if (BGP_DEBUG (events, EVENTS))
+      {
+          zlog_debug("BGP stop: %s (dynamic neighbor) deleted", peer->host);
```

Comment:
-> zlog_info

check line 467: neighbor down event is logged as info level, this is also neighbor down event, should also logged as info level.